sympy/parsing/tests/test_latex.py

```diff
@@ -187,7 +193,12 @@ def test_import():
 def test_parseable():
     from sympy.parsing.latex import parse_latex
     for latex_str, sympy_expr in GOOD_PAIRS:
-        assert parse_latex(latex_str) == sympy_expr
+        try:
+            assert parse_latex(latex_str) == sympy_expr
+        except AssertionError:
+            print(latex_str)
+            print(sympy_expr)
```

Comment:
I don't think that it's good idea to use print, because it can mess up with the test runner progress printing.
But rather wrap the error messages in the exceptions. For example, you can do like `raise AssertionError(latex_str)` to throw the error again

Reply:
I think I understand what you mean. Please let me know if my implementation is not what you intended.